lib/full-site-editing/block-templates.php

```diff
@@ -105,10 +105,20 @@ function _gutenberg_build_template_result_from_file( $template_file, $template_t
  *
  * @param WP_Post $post Template post.
  *
- * @return WP_Block_Template Template.
+ * @return WP_Block_Template|WP_Error Template.
  */
 function _gutenberg_build_template_result_from_post( $post ) {
-	$theme = get_the_terms( $post, 'wp_theme' )[0]->slug;
+	$terms = get_the_terms( $post, 'wp_theme' );
+
+	if ( is_wp_error( $terms ) ) {
```

Comment:
Shouldn't we handle the error here, instead of returning `$terms` and check again after the call of `_gutenberg_build_template_result_from_post`?

Reply:
What do  you mean by "handle" here? The handling is  actually "ignoring the template" in the caller.

Reply:
I was thinking about throwing but seems it's not what we want here..